src/qt/res/README.md

```diff
@@ -0,0 +1,46 @@
+# Icon policy
+This document provides guidelines for contributing icons to Bitcoin Core.
+
+## Preparing Icons
+Both an icon source file, in Scalable Vector Graphics (SVG) format, and an optimized production file, in Portable Network Graphics (PNG) format, are required for each icon.
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality but are not used in production due to limited application support.
+If different-sized production (PNG) icons are required, they can be generated from the associated SVG source file in any vector-based tool.
+
+#### PNG Production File
+PNGs are used in production due to wide application support, transparency support, and better image quality compared to competing file types
+such as JPEG.
+
+### Icon Grid
+Bitcoin Core uses an 8-point grid system for its icons which utilizes size increments of 8px to visually orientate elements within a grid. The majority of screen sizes are divisible by 8, which makes fitting elements easier using an 8-point grid system. This system makes things more efficient for designers and aids in maintaining consistency across applications.

```

Comment:
```suggestion

```

Reply:
You no longer want this section in there?

Reply:
Nah these details are all in the Bitcoin icons repo.